.github/workflows/foundry.yml

```diff
@@ -10,6 +10,9 @@ on:
   push:
     branches:
       - "main"
+    paths:
```

Comment:
not really related to this, so can take it out. I just figure it would be good limit runs of this when it's not necessary.